trackintel/model/trips.py

```diff
@@ -101,13 +145,70 @@ def temporal_tracking_quality(self, *args, **kwargs):
 
         See :func:`trackintel.analysis.tracking_quality.temporal_tracking_quality`.
         """
-        return ti.analysis.tracking_quality.temporal_tracking_quality(self._obj, *args, **kwargs)
+        return ti.analysis.tracking_quality.temporal_tracking_quality(self, *args, **kwargs)
 
-    def generate_tours(self, *args, **kwargs):
+    def generate_tours(self, **kwargs):
         """
         Generate tours based on trips (and optionally staypoint locations).
 
         See :func:`trackintel.preprocessing.trips.generate_tours`.
         """
-        assert len(args) == 0, "When calling 'generate_tours' via the accessor all arguments must be keyword arguments"
-        return ti.preprocessing.trips.generate_tours(trips=self._obj, **kwargs)
+        return ti.preprocessing.trips.generate_tours(trips=self, **kwargs)
+
+
+# added GeoDataFrame and DataFrame manually afterwards such that our methods always come first
+class TripsGeoDataFrame(TrackintelGeoDataFrameWithFallback, TripsDataFrame, gpd.GeoDataFrame, pd.DataFrame):
+    """Class to treat a GeoDataFrame as collections of trips.
+
+    Requires at least the following columns:
+    ['user_id', 'started_at', 'finished_at', 'origin_staypoint_id', 'destination_staypoint_id']
```

Comment:
shouldn't the geometry be a required column here?

Reply:
So far we did not included `geometry` as required column in the other classes as well.
But I adapted the information of `MultiPoint` some lines further down, such that it is clear that a geometry is required.